_posts/2021-04-28-cockpit-243.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Cockpit 243
+author: pitti
+date: 2021-04-28
+tags: cockpit linux
+slug: cockpit-243
+category: release
+summary: Cockpit with Services page improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 243.
+
+## Services: Show sockets and memory usage
+
+Socket units show listening paths and ports as well as service units they activate.
```

Comment:
So an extra word is not needed before service, but having any or the makes it clear service units is a noun, not "service (verb) units".

```suggestion
Socket units show active paths and ports, as well as any service units they activate.
```
```suggestion
Socket units show active paths and ports, as well as the service units they activate.
```

I think any works better here, but the also works. (I might be wrong, and I don't care which one you pick.)

It's also a bit difficult to read as it feels like it bounces around and keeps going on, so I added a comma. I might be wrong about this too. But "x and y as well as z" feels like a roller coaster.